code/modules/mining/equipment/kinetic_crusher/trophies/lavaland_mob_trophies.dm

```diff
@@ -0,0 +1,208 @@
+	bonus_value = 3
+
+/obj/item/crusher_trophy/legion_skull/effect_desc()
+	return "a kinetic crusher to recharge <b>[bonus_value*0.1]</b> seconds faster"
+
+/obj/item/crusher_trophy/legion_skull/add_to(obj/item/kinetic_crusher/target_crusher, mob/living/user)
+	. = ..()
+	if(.)
+		target_crusher.charge_time -= bonus_value
+
+/obj/item/crusher_trophy/legion_skull/remove_from(obj/item/kinetic_crusher/target_crusher, mob/living/user)
+	. = ..()
+	if(.)
+		target_crusher.charge_time += bonus_value
+
+/**
+ * Lobstrosity
+ * Detonating a mark causes the victim to stagger for 1 second, which slows down movement
+ * and more than doubles the cooldown for the victim's ranged attack (if it's a simple hostile mob).
+ */
+/obj/item/crusher_trophy/lobster_claw
+	name = "lobster claw"
+	icon_state = "lobster_claw"
+	desc = "A lobster claw. Suitable as a trophy for a kinetic crusher."
+	denied_type = /obj/item/crusher_trophy/lobster_claw
+	bonus_value = 1 SECONDS
+
+/obj/item/crusher_trophy/lobster_claw/effect_desc()
+	return "mark detonation to briefly stagger the target for <b>[bonus_value * 0.1]</b> seconds"
+
+/obj/item/crusher_trophy/lobster_claw/on_mark_detonation(mob/living/target, mob/living/user)
+	target.apply_status_effect(/datum/status_effect/stagger, bonus_value)
+
+/**
+ * Brimdemon
+ * Detonating a mark causes a loud = funny sound and a comedic message to appear on the victim.
+ */
+/obj/item/crusher_trophy/brimdemon_fang
+	name = "brimdemon's fang"
+	icon_state = "brimdemon_fang"
+	desc = "A fang from a brimdemon's corpse. Suitable as a trophy for a kinetic crusher."
+	denied_type = /obj/item/crusher_trophy/brimdemon_fang
+	///Humorous words to display on mark detonation
+	var/static/list/comic_phrases = list("BOOM", "BANG", "KABLOW", "KAPOW", "OUCH", "BAM", "KAPOW", "WHAM", "POW", "KABOOM")
+
+/obj/item/crusher_trophy/brimdemon_fang/effect_desc()
+	return "mark detonation to create <b>visual and audiosensory effects</b> on the target"
+
+/obj/item/crusher_trophy/brimdemon_fang/on_mark_detonation(mob/living/target, mob/living/user)
+	var/turf/victim_turf = get_turf(target)
+	victim_turf.balloon_alert_to_viewers("[pick(comic_phrases)]!")
+	playsound(victim_turf, 'sound/lavaland/brimdemon_crush.ogg', 100)
+
+/**
+ * Bileworm
+ * Detonating a mark causes 4 acid spewlets to be shot in 4 cardinal directions, on a 10 seconds cooldown.
+ */
+/obj/item/crusher_trophy/bileworm_spewlet
+	name = "bileworm spewlet"
+	icon = 'icons/mob/simple/lavaland/bileworm.dmi'
+	icon_state = "bileworm_spewlet"
+	desc = "A baby bileworm. Suitable as a trophy for a kinetic crusher."
+	denied_type = /obj/item/crusher_trophy/bileworm_spewlet
+	///item ability that handles the effect
+	var/datum/action/cooldown/mob_cooldown/projectile_attack/dir_shots/spewlet/ability
+
+/obj/item/crusher_trophy/bileworm_spewlet/effect_desc()
+	return "mark detonation to <b>launch projectiles</b> in cardinal directions on a <b>10</b> second cooldown"
```

Comment:
The reason that I think that `bonus_value` is pointless is that people stop bothering to use it

Reply:
this trophy is a sin and armhulen knew that when making it